### ISSUE TITLE
Hi, I would like to ask you about the quantization part inside the code (is there a paper and formula?) BinOp()

### ISSUE THREAD

**Issue by liyanjie001** [Intelligent-Computing-Lab-Yale/SpikeSim#1]
Hi, I would like to ask you about the quantization part inside the code (is there a paper and formula?) BinOp()

**Comment by MoitraAbhi** [comment]
Hi, BinOp() has been adapted from the quantization technique proposed in https://github.com/jiecaoyu/XNOR-Net-PyTorch based on the paper XNOR-Net: ImageNet Classification Using Binary Convolutional Neural Networks.

**Comment by liyanjie001** [comment]
Hi, when trained with this framework of yours, the test dataset inference accuracy is also very low around 40%. (quantization-bits set to 1 and 4, timestep set to 5, coding method: direct and Poisson coding, epoch: 100, Arch: VGG9, dataset: cifar10) Do you have any guidelines to use your framework to achieve better training results and acceptable inference accuracy

**Comment by liyanjie001** [comment]
Hi, when trained with this framework of yours, the test dataset inference accuracy is also very low around 40%. (quantization-bits set to 1 and 4, timestep set to 5, coding method: direct and Poisson coding, epoch: 100, Arch: VGG9, dataset: cifar10) Do you have any guidelines to use your framework to achieve better training results and acceptable inference accuracy

**Comment by MoitraAbhi** [comment]
Hi we recommend using the default settings of train.py. With lr = 0.001 (for direct coding) the accuracy should be high. Feel free to reach out to us if any other problem persists.
